index.html

```diff
@@ -9,72 +9,75 @@
         <link rel="preconnect" href="https://fonts.googleapis.com">
         <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
         <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter:wght@400;500;700&family=Poppins:wght@400;700&family=Roboto:wght@400;700&display=swap" rel="stylesheet">
+        <script defer src="index.js"></script>
     </head>
     <body>
-        <header class="nav">
-            <a href="#" class="nav-title">Innocent N.</a>
-            <nav class="nav-nav">
-                <img src="./images/menu.png" alt="menu-icon" class ="nav-menu" />
-                <ul class="links">
-                    <li>
-                        <a href="#">Portfolio</a>
-                    </li>
-                    <li>
-                        <a href="#">About</a>
```

Comment:
- [ ] It will be better if you try to remove the green border from this class ```.new-container``` to make the design like the Figma 😃

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Figma design | Your design| 
| --- | --- |
| ![image](https://user-images.githubusercontent.com/83360636/224975403-0fd52bc6-ff82-4616-adae-28fb856be1a5.png)  |    ![image](https://user-images.githubusercontent.com/83360636/224975340-caf33647-0d70-4826-8fad-34ba714689b0.png)  |